src/Postnl.php

```diff
@@ -238,15 +244,18 @@ public function generateBarcodeByDestination(
         $customerNumber = null,
         $serie = null
     ) {
+        $eps = false;
```

Comment:
It's probably sufficient to put `$eps = $countryCode != 'NL'` here. And leave an empty row below it.